packages/office-ui-fabric-react/src/components/Layer/examples/Layer.Customized.Example.tsx

```diff
@@ -1,25 +1,22 @@
-import * as React from 'react';
 import { Customizer } from '@uifabric/utilities';
-import { Panel } from 'office-ui-fabric-react/lib/Panel';
 import { Checkbox } from 'office-ui-fabric-react/lib/Checkbox';
 import { LayerHost } from 'office-ui-fabric-react/lib/Layer';
+import { Panel } from 'office-ui-fabric-react/lib/Panel';
+import * as React from 'react';
 
-export interface ILayerCustomizedExampleState {
+interface IState {
   showPanel: boolean;
   trapPanel: boolean;
 }
 
-export class LayerCustomizedExample extends React.Component<{}, ILayerCustomizedExampleState> {
-  constructor(props: {}) {
-    super(props);
-
-    this.state = {
-      showPanel: false,
-      trapPanel: false
-    };
-  }
+export class LayerCustomizedExample extends React.Component<{}, IState> {
+  // tslint:disable-next-line:member-access
```

Comment:
State is not private in the React.Component, however the linting rules are too blunt (maybe that can't be changed) and so it wants member access to be defined for state, I think this rule is too strict here.

Reply:
We usually define state as public. Can you change it to public and remote the tslint:disable? (Somehow I missed this one earlier, sorry--in the future, it's probably best to leave comments that you want others to see unresolved until late in the review process.)

Reply:
Apologies, I had fixed this and resolved, but didn't comment that I fixed.

Reply:
No worries, looking back at this now I realize that I forgot to look at the latest version (instead of the old one where the comment was made). This is a major reason I'm not a fan of GitHub's PR review interface... :(